.github/workflows/validate-pr-title.yaml

```diff
@@ -0,0 +1,38 @@
+---
+name: "Validate PR Title"
+# Ensure that the PR title conforms to the Conventional Commits and our choice of types and scopes, so that library version bumps can be detected automatically
+
+on:
+  pull_request_target:
+    types: [opened, edited, synchronize]
+
+permissions:
+  pull-requests: read
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  main:
+    name: Validate PR title
+    runs-on: ubuntu-latest
+    steps:
+      - uses: amannn/action-semantic-pull-request@v5
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          types: |
+            chore
+            ci
+            docs
+            feat
+            fix
+            perf
+            refactor
+            revert
+            test
+          scopes: |
+            harness
+            [a-z]{3,20}
```

Comment:
Shouldn't we prevent any other scope than `harness`? The PR title should leave it off in that case.

Reply:
Let's chat at next standup.
I thought we'd want to allow free-form scope, for:
- outstanding #1148 
- upcoming work like `feat(notice)`
- hypothetical `feat(immutable)` work

I'm happy to go with whatever team decides, this is meant to be pragmatic after all.

Reply:
Decided at the stand-up: remove free-form scope.